packages/iov-tendermint-rpc/src/client.spec.ts

```diff
@@ -29,20 +30,29 @@ function pendingWithoutTendermint(): void {
  * in the port 111<version>, e.g. Tendermint 0.21.0 runs on port 11121. To start
  * a legacy version use
  *   TENDERMINT_VERSION=0.21.0 TENDERMINT_PORT=11121 ./scripts/tendermint/start.sh
+ *   TENDERMINT_VERSION=0.25.0 TENDERMINT_PORT=11125 ./scripts/tendermint/start.sh
+ *   TENDERMINT_VERSION=0.27.0 TENDERMINT_PORT=11127 ./scripts/tendermint/start.sh
  *
  * When more than 1 instances of tendermint are running, stop them manually:
  *   docker container ls | grep tendermint/tendermint
  *   docker container kill <container id from 1st column>
  */
 const tendermintInstances = [
-  // {
-  //   url: "localhost:11121",
-  //   version: "0.21.x",
-  // },
   {
     url: "localhost:12345",
     version: "0.25.x",
+    appCreator: "jae",
   },
+  // {
```

Comment:
Commented out?
This looks like a great config with appCreator and all

Reply:
The stop script only supports one container labeled "tendermint/tendermint" right now. I think it is out of scope to run both in the CI. It can be done, but it is some work.

What I commented out is for local development. It works fine!